client/layout/universal-navbar-footer/index.tsx

```diff
@@ -12,6 +13,8 @@ const UniversalNavbarFooter = () => {
 	const { shouldSeeDoNotSell, isDoNotSell, onSetDoNotSell } = useDoNotSell();
 	const [ isDialogOpen, setIsDialogOpen ] = useState( false );
 
+	const realPathName = getRealPathName( window.location.pathname.slice( 1 ) );
```

Comment:
This  (and line 213) breaks SSR rendering (and corresponding e2e tests) because we don't have a `window` object on the server-side.
An alternative may be to use the `getCurrentRoute` selector instead.